tasks/metrics/mutant.rake

```diff
@@ -33,9 +33,13 @@ namespace :metrics do
       --use #{config.strategy}
     ].concat(ignore_subjects).concat(namespaces).concat(since)
 
-    status = namespace::CLI.run(arguments)
-    if status.nonzero?
-      Devtools.notify_metric_violation 'Mutant task is not successful'
+    result = namespace::CLI.run(arguments)
+
+    # Mutant 0.8.8 returns a Fixnum while 0.8.9 returns a boolean
```

Comment:
We could always bump the devtools mutant dep to be 0.8.9 so this extra conditional isn't required.